tests/integration_tests/tests/test_cases.py

```diff
@@ -124,8 +125,10 @@ def _save_manager_logs_after_test(self, purge=True):
         self.logger.debug('opening tar.gz: {0}'.format(target))
         with tarfile.open(target) as tar:
             tar.extractall(path=logs_dir)
-        self.logger.debug('removing {0}'.format(target))
-        os.remove(target)
+        if not bool(os.environ.get('DONT_REMOVE_LOG_TAR')):
```

Comment:
it's better not to extract to tar for the first place, because it's a waste of time in that case

Reply:
Change to `SKIP_LOG_SAVE_ON_SUCCESS`